Enterprise-CMCS/macpro-onemac
# OY2 23845: Withdraw Formal RAI Response - Email

Story: https://qmacbis.atlassian.net/browse/OY2-23845
Endpoint: https://d1jfwoxkadkgby.cloudfront.net

### Details
This feature creates the CMS notice and the State receipt for the RAI Response withdraw action.

### Changes
- updated package builder to include the cpocEmail and the reviewTeamEmails in the package item
- added the State and CMS Withdraw RAI response emails
- updated the CMS email creation to include the async lookup of the cpoc and srt emails
- because the current withdraw rai reponse form is type-agnostic, added a type-check to determine which CMS emails are included.
- Updated seed data to better support automated testing
- Updated resetData to better support automated testing.

### Test Plan
1. Find a way to see/be included in the emails being sent.  (using different ways to verify might uncover missed bugs/acs)
2. Login as a State user
3. Navigate to dashboard
4. perform 5-9 for each of: Medicaid SPA, CHIP SPA, Initial B waiver, B waiver Renewal, B waiver Amendment, C Waiver "App K" amendment.
5. Locate a Medicaid SPA in the "RAI Response Withdraw Enabled"
6. Begin the "Withdraw Formal RAI Response" action
7. Submit the form
8. Verify the state receipt is sent to the correct parties and looks as described in the story AC 
9. Verify the CMS notice is sent to the correct parties and looks as described in the story AC
